src/App.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-
+import BubbleButton from './components/BubbleButton';
```

Comment:
please reset App.js after testing your component. this button won't be directly inside app.js